tools/ci.psm1

```diff
@@ -263,21 +277,48 @@ function Update-TestResults
     }
 }
 
+function Invoke-CIxUnit
+{
+    param(
+        [switch]
+        $SkipFailing
+    )
+    $env:CoreOutput = Split-Path -Parent (Get-PSOutput -Options (Get-PSOptions))
+    if(!(Test-Path "$env:CoreOutput\pwsh.exe"))
```

Comment:
Can we change this to work on Linux and macOS

Reply:
In the PR xUnit tests run only on Windows. Previously we run the tests on Unix-s too.

Reply:
This PR doesn't change Linux or macOS.  I'll deal with this issue when refactoring macOS and Linux